cfpb/cfgov-refresh
# Add CCDB Trend Libraries

_Updates the Consumer Complaint Database UI and API to the latest production version, adding some new features._


---

## Changes

- Version bumps for CCDB5-UI and CCDB5-API library packages


## How to test this PR

1. Run the application. CCDB5 .whl packages should properly install.
1. If you enable the "CCDB5_TRENDS" flag, you will see a "Trends" tab appear on CCDB Search.
1. A /trends endpoint will be available from a production URL.

## Checklist

  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing with the feature/section you're addressing, e.g., "Mega Menu: fix layout bug" or "Paying for College: Update content on Repay tool"

### Front-end testing
This testing occurs within the CCDB5-UI repository
